src/stylesheets/components/_background-layout.scss

```diff
@@ -1,25 +1,27 @@
 .background-layout {
   &__image {
+    position: absolute;
```

Comment:
Does this needs to be absolute? Maybe setting it to relative and the overall would be sufficient.

Also when using `position: absolute;`, there needs to be `position: relative` set in the parent class (the component root) to make sure that the positioning is not affecting beyond the boundaries of the component.

Finally, also set `z-index` when using `position: absolute;` to avoid nasty surprises. Without it, the styles will rely on the ordering of the code (last rules overrides everything else) which is not always what you want.

Reply:
The `position` isn't needed as you said. I was blindly following figma that time 😐 making the component `100vh` and `100vw` and setting the image's `height` and `width` to 100% makes it work easily. Updated in https://github.com/rafayet-monon/surveyor/pull/13/commits/85d52aa97d04c895ad920aa3419db76cde1ba749